src/components/atoms/frame/Modal.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { type ReactNode } from "react";
+import Button from "../buttons/Button";
+import { IoIosCloseCircle } from "react-icons/io";
+
+type Props = {
+  titleTextSize?: string;
+  buttonHeight?: string;
+  buttonWidth?: string;
+  open: boolean;
+  setOpen: React.Dispatch<React.SetStateAction<boolean>>;
+  onDoneClick?: () => void;
+  className?: string;
+  title: string;
+  children?: ReactNode;
+  closeOnClick?: () => void;
+  isDoneButton?: boolean;
+  doneButtonText: string;
+  isCancelButton?: boolean;
+  cancelButtonText?: string;
+  minWidthModal?: string;
+  crossOnClick?: () => void;
+};
+
+export const Modal = (props: Props) => {
+  const {
+    open,
+    className,
+    buttonHeight,
+    buttonWidth,
+    children,
+    title,
+    isDoneButton = true,
+    doneButtonText,
+    onDoneClick,
+    closeOnClick,
+    isCancelButton = false,
+    cancelButtonText,
+    titleTextSize,
+    crossOnClick,
+  } = props;
+
+  if (!open) return null;
+
+  return (
+    <div className={`modal modal-box ${open && "modal-open"}`}>
+      <div className="fixed inset-0 flex items-center justify-center bg-black bg-opacity-50 text-[#1D212D]">
+        <div
+          className={`relative bg-[#ECECEC] rounded-xl shadow gap-[14px] flex flex-col justify-between ${className}`}
+        >
+          <IoIosCloseCircle
+            className="absolute top-3 right-3 text-2xl text-gray-500 cursor-pointer hover:text-primary"
+            onClick={crossOnClick || closeOnClick}
+          />
+
+          <span
+            className={`px-6 flex w-full flex-row justify-between font-bold ${
+              titleTextSize ? titleTextSize : "text-[20px] pt-2 font-bold"
+            }`}
+          >
+            {title}
+          </span>
```

Comment:
it's not good syntax to use absolute for a simple positioning .  rather this syntax is common 
```  <div className="flex justify-between items-center border">
            <span
              className={`px-6 flex w-full flex-row justify-between font-bold ${
                titleTextSize ? titleTextSize : "text-[20px] pt-2 font-bold"
              }`}
            >
              {title}
            </span>
            <IoIosCloseCircle
              className="right-3 text-2xl text-gray-500 cursor-pointer hover:text-primary"
              onClick={crossOnClick || closeOnClick}
            />
          </div>
          
```